static/css/app.css

```diff
@@ -807,11 +821,27 @@
 /* -------------------------------------------------------------------------- */
 
 .schema {}
-.schema i                           { display: inline-block; margin-right: 4px; }
-.schema i.fa-folder-o               { display: inline-block; }
-.schema i.fa-folder-open-o          { display: none; }
-.schema.expanded i.fa-folder-open-o { display: inline-block; }
-.schema.expanded i.fa-folder-o      { display: none; }
+
+.schema i {
```

Comment:
I would prefer you keeping the original formatting where possible

Reply:
Sure, I've reverted the auto-formatted code to maintain the original formatting